src/js/base/Label.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme } from "@app/theme";
+import { getBadgeOrLabelColor } from "@base/utils";
+import React from "react";
+
+type LabelProps = {
+    children?: React.ReactNode;
+    className?: string;
+    color?: string;
+};
+
+/**
+ * A styled Label component
+ */
+export function Label({ children, className, color }: LabelProps) {
+    return (
+        <span
+            className={`text-[${theme.color.white}] rounded-[${theme.borderRadius.sm}] bg-[${getBadgeOrLabelColor({
+                color,
+            })}] inline whitespace-nowrap text-center font-bold px-2 py-1 text-[${
+                theme.fontSize.sm
+            }] ${className} align-baseline last-of-type:m-0 `}
+        >
+            {children}
+        </span>
```

Comment:
* Just use the Tailwind classes like `text-white`, `rounded-sm` directly.
* Use something like `cslx` or `classnames` packages for conditional styles. Try `cslx` first if one isn't already installed.